pyvisa-py/protocols/rpc.py

```diff
@@ -429,11 +456,7 @@ def make_call(self, proc, args, pack_func, unpack_func):
             pack_func(args)
         call = self.packer.get_buf()
         self.sock.sendto(call, (self.host, self.port))
-        try:
```

Comment:
First off, thanks for much for giving this a look over @MatthieuDartiailh. :+1:

As for why it was done this way originally, I can't say for sure, but I can think of two possible reasons:

1. To shorten the call (`select(r, w, x)` instead of `select.select(r, w, x)`)
2. To gracefully allow for the possibility that `select` could fail to be imported and continue with the warning that the socket hanging is a very real possibility. That said, I can't see why this would _ever_ happen here. `select.select()` has been in the Python core modules since 2.5, and neither pyvisa nor pyvisa-py claim to formally support anything below Python 2.6.

Since I ended up needing it in `do_call()` and I tend to prefer the absolute call style (especially if they have generic names), I figured it was justified to just import at the module level and use it the same everywhere.